### DISCUSSION TITLE
Bagaimana membaca Plot MSTS

### DISCUSSION THREAD

**Discussion by intanmsari** [teamalgoritma/community#897] [Q&A] [answered]
![Screenshot 2024-08-06 152908](https://github.com/user-attachments/assets/4efe9dfc-623e-4ec9-a5b8-34c808aa7658)

mohon bantuannya, bagaimana cara membaca plot dari msts ini ya?

**Comment by saskia-dwi-ulfah** [comment, marked as answer]
Dear Bu @intanmsari ,
Terima kasih telah bertanya melalui Github Discussion Algoritma, 

Plot hasil `mstl()` yang ibu lampirkan di atas menunjukkan hasil decompose dari data time series yang dimiliki. Dari plot di atas, data asli kita (pada plot `Data`) akan diuraikan menjadi komponen `Trend`, `Seasonal13` (yang menunjukkan pola perulangan harian, daily seasonality), `Seasonal91` (yang menunjukkan pola perulangan mingguan).

Adapun sumbu x-axis menunjukkan indeks waktu. Karena ibu melakukan decompose dengan seasonality harian (13 * 7) dan mingguan (13 * 7 *4) saat mendefinisikan `msts()`, maka x-axis menunjukkan waktu "minggu ke". Misalnya, pada gambar di atas, ibu melakukan decompose untuk data 1 bulan (pada sintaks `head(13 * 7* 4)`). Saat angka di sumbu x adalah 1, maka itu menunjukkan awal dari minggu pertama. Plot akan berhenti sebelum angka 5 pada sumbu x  (menunjukkan akhir minggu ke 4).


Agar lebih terlihat perbedaannya, saya juga coba melampirkan contoh apabila hanya mendefinisikan seasonality harian pada `msts()`. Interpretasi pada saat membaca x-axis adalah "hari ke". Kemudian proses decompose dilakukan pada data 1 bulan (pada sintaks `head(13 * 7* 4)`). Pada x-axis, plot akan berhenti sebelum angka 30 (menunjukkan hari terakhir dalam 1 bulan)
![image](https://github.com/user-attachments/assets/6dbd1562-54e0-4212-b315-f4422a7f5b80)

Semoga jawaban ini dapat membantu,

Best Regards
Team Algoritma

**Comment by adizaen** [comment]
Dear Bu @intanmsari 

Untuk menjaga agar forum diskusi berjalan kondusif, karena tidak ada respon lanjutan setelah 7 hari dari respon terakhir, kami izin menandai pertanyaan sudah terjawab dan menutup diskusi ini. Semoga dapat dipahami dan terima kasih.

Best regards,
**Team Algoritma**
